src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,184 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import Image from "next/image";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { Button } from "../../../../../../../components/server/Button";
+import Link from "next/link";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { ClockIcon } from "../../../../../../../components/server/Icons";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "PIN";
+  breachData: GuidedExperienceBreaches;
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  const highRiskDataBreaches = props.breachData.highRisk;
+  let exposedData: SubscriberBreach[] = [];
+
+  if (props.breachData) {
+    switch (props.typeOfBreach) {
+      case "ssnBreaches":
+        exposedData = highRiskDataBreaches.ssnBreaches;
+        break;
+      case "creditCard":
+        exposedData = highRiskDataBreaches.creditCardBreaches;
+        break;
+      case "bankAccount":
+        exposedData = highRiskDataBreaches.bankBreaches;
+        break;
+      case "PIN":
+        exposedData = highRiskDataBreaches.pinBreaches;
+        break;
+      default:
+        break;
+    }
+  }
+
+  // TODO: Make locale location-sensitive in the future
+  const dateFormatter = new Intl.DateTimeFormat("en-US", {
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle
+    dateStyle: "short",
+  });
+
+  const listOfBreaches = exposedData.map((item: SubscriberBreach) => (
+    <div key={item.id} className={styles.breachItem}>
+      {item.name}
+      <span className={styles.date}>{` on ${dateFormatter.format(
+        new Date(item.addedDate)
+      )}`}</span>
```

Comment:
nit: not sure what we want to do about l10n stuff here. We're hardcoding a few English strings that will will need to be localized eventually (ie: L68, L73-76, L83-87, L92-94, L102-104, etc)